lms/envs/bok_choy.env.json

```diff
@@ -80,7 +80,6 @@
         "ENABLE_DISCUSSION_SERVICE": true,
         "ENABLE_S3_GRADE_DOWNLOADS": true,
         "ENABLE_THIRD_PARTY_AUTH": true,
-        "ENABLE_COMBINED_LOGIN_REGISTRATION": true,
```

Comment:
You should open a corresponding PR against https://github.com/edx/configuration to remove this from the edxapp role's defaults.